projects/packages/jetpack-mu-wpcom/src/features/wpcom-sidebar-notice/wpcom-sidebar-notice.php

```diff
@@ -38,14 +38,12 @@ function wpcom_enqueue_sidebar_notice_assets() {
 		$asset_file['version'] ?? filemtime( Jetpack_Mu_Wpcom::BASE_DIR . 'build/wpcom-sidebar-notice/wpcom-sidebar-notice.css' )
 	);
 
-	wp_localize_script(
-		'wpcom-sidebar-notice',
-		'wpcomSidebarNoticeConfig',
-		array(
-			'ajaxUrl' => admin_url( 'admin-ajax.php' ),
-			'nonce'   => wp_create_nonce( 'wpcom_fetch_sidebar_notice' ),
-		)
+	$data          = array(
+		'ajaxUrl' => admin_url( 'admin-ajax.php' ),
+		'nonce'   => wp_create_nonce( 'wpcom_fetch_sidebar_notice' ),
 	);
+	$inline_script = 'const wpcomSidebarNoticeConfig = ' . wp_json_encode( $data ) . ';';
```

Comment:
Nitpick: We could include the array inline without assigning that to a variable.